src/pages/todos/TodosPage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const TodosPage = () => {
+  return (
+    <div>Todos Page</div>
```

Comment:
This component should hold several components, in my opinion :
The todos list, which would map over all todos. Each todo item should be a component then (with the dropdown button and all). and there is also the top nav which holds the title and the profile. These are all components.
I had a rationale when I formed these folders, of course it's absolutely fine that you changed it, I just don't follow the entire logic this new structure...

Reply:
We just needed the basic router + folder structure for the beginning. No todos logic yet. 
Of course each todo item will be a component by itself, I didn't get into that just yet. But it is written on the todos page on trello:
2. List of todos, => TodoItem component

The navbar is not specific for the todos, the same nav will render in the user info.

Reply:
I don't know if there's a separate page for user info, this is the only page the nav will be rendered, if the user info is a pop-up / modal... This nav bar doesn't really navigate..

Reply:
It could be a pop-up modal:) or a separate page. Anyway, the todo list will be rendered in the todos page, importing to it and rendering also todo item. The navbar should not be inside a specific page as a good practice. You want it above all the pages:)